e2e/volume_ops_test.go

```diff
@@ -79,6 +82,29 @@ func testVolumeCreate(t *testing.T) {
 	r.Nil(errVolCreate)
 }
 
+func testVolumeExpand(t *testing.T) {
+	r := require.New(t)
+
+	var brickPaths []string
+	for i := 1; i <= 4; i++ {
+		brickPath, err := ioutil.TempDir(tmpDir, "brick")
+		r.Nil(err)
+		brickPaths = append(brickPaths, brickPath)
+	}
+
+	expandReq := api.VolExpandReq{
+		ReplicaCount: 2,
+		Bricks: []string{
+			gds[0].PeerID() + ":" + brickPaths[0],
+			gds[1].PeerID() + ":" + brickPaths[1],
+			gds[0].PeerID() + ":" + brickPaths[2],
+			gds[1].PeerID() + ":" + brickPaths[3],
```

Comment:
You're using the exact same bricks as the ones that were used to create the volume. So ideally this would be a negative test case.
Keep the negative case, and add a new positive case, with new bricks.

Reply:
But doesn't ioutil.TempDir create new temporary directories each time?

Reply:
You're right. My bad.